SetupDataPkg/ConfApp/ConfApp.h

```diff
@@ -9,11 +9,9 @@
 #define CONF_APP_H_
 
 #include <UefiSecureBoot.h>
-#include <DfciSystemSettingTypes.h>
-#include <Protocol/DfciAuthentication.h>
-#include <Protocol/DfciSettingAccess.h>
 
-#define NO_TRANSITION_STATE  MAX_UINTN
+#define NO_TRANSITION_STATE    MAX_UINTN
+#define DEFAULT_USB_FILE_NAME  L"SetupConfUpdate.svd"

```

Comment:
Would this be better as a PCD?  Asking because if someone is supporting multiple projects, they may want to be able to have something specific to their project...

Reply:
Thanks for the suggestion. This is updated to be using `PcdConfigurationFileName`.